### PR TITLE
Continue port of QueueNodeHearbeat

### PR DESCRIPTION
**Pull request by chkeita**
Inject the logTracer
partial ports of Events, webhooks and queue

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/NodeOperations.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        :base(storage)
+    {
+        
+    }
+
+    public async Task<Node?> GetByMachineId(Guid machineId)
+    {
+        var tableClient = await GetTableClient("Node");
+
+        var data = QueryAsync(filter: $"RowKey eq '{machineId}'");
```
Comment (nharper285) [inline src/ApiService/ApiService/onefuzzlib/NodeOperations.cs:29]: How is tableClient used? It looks like you're just passing the filter to QueryAsync, but no actual reference to the table.

File: src/ApiService/ApiService/QueueNodeHearbeat.cs [hunk 2]
```diff
@@ -14,20 +14,25 @@ namespace Microsoft.OneFuzz.Service;
...
 
     [Function("QueueNodeHearbeat")]
     public async Task Run([QueueTrigger("myqueue-items", Connection = "AzureWebJobsStorage")] string msg)
     {
+        _logger.LogInformation($"heartbeat: {msg}");
+
         var hb = JsonSerializer.Deserialize<NodeHeartbeatEntry>(msg, EntityConverter.GetJsonSerializerOptions()).EnsureNotNull($"wrong data {msg}");
 
-        var node = await Node.GetByMachineId(_storageProvider, hb.NodeId);
+        var node = await _nodes.GetByMachineId(hb.NodeId);
```
Comment (tevoinea) [inline src/ApiService/ApiService/QueueNodeHearbeat.cs:35]: That looks really nice

File: src/ApiService/ApiService/onefuzzlib/Events.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.OneFuzz.Service
+{
+
+    public record SignalREvent
+    (
+        string Targe,
```
Comment (tevoinea) [inline src/ApiService/ApiService/onefuzzlib/Events.cs:13]: Targe**t**?

File: src/ApiService/ApiService/onefuzzlib/NodeOperations.cs (hunk 1 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/NodeOperations.cs:27]: yes not needed. Removed.

**Pull request merged**